src/views/InteractionDesigner.vue

```diff
@@ -55,7 +55,7 @@ import {store} from '@/store'
 // import TreeViewer from './TreeViewer'
 // import LegacyInteractionDesigner from './InteractionDesigner.legacy'
 // import TreeUpdateConflictModal from './TreeUpdateConflictModal';
-import TreeBuilderToolbar from '@/components/interaction-designer/toolbar/TreeBuilderToolbar.vue'
+// import TreeBuilderToolbar from '@/components/interaction-designer/toolbar/TreeBuilderToolbar.vue'
```

Comment:
Thanks @object-required!
I've cleaned up the commented-out lines now.
@safydy-r @jcbashdown Let me know if you'd like them back.

Reply:
Good to remove them I reckon! I think they stuck around in case Brett had something in mind. Now we're back to properly working on this repo we can choose our own path.